app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt

```diff
@@ -124,14 +126,11 @@ class EditKeyboardFragment : BaseFragment() {
         binding?.backButton?.action = {
             val isEditing = arguments?.getBoolean(KEY_IS_EDITING) ?: false
             val textChanged = binding?.keyboardInput?.text.toString() != phrase?.getLocalizedText()
-            if (isCategory) {
+            val categoryTextChanged = binding?.keyboardInput?.text.toString() != category?.getLocalizedText()
```

Comment:
Are you changing `keyboard_input` to an `AppCompatTextView` here or in another PR?

Reply:
another PR :]